pinecone/core/grpc/index_grpc.py

```diff
@@ -171,9 +171,10 @@ def parse_fetch_response(response: dict):
     return FetchResponse(vectors=vd, namespace=namespace, _check_type=False)
 
 
-def parse_query_response(response: dict, unary_query: bool):
+def parse_query_response(response: dict, unary_query: bool, _check_type: bool = False):
     res = []
 
+    # TODO: consider deleting this deprecated case
     for match in response.get('results', []):
         namespace = match.get('namespace', '')
         m = []
```

Comment:
arent we removing this?

Reply:
Only if we can remove the `unary_query` param as well, which I wasn't sure about.  
I preferred not to introduce accidentally breaking changes to the coming release.

Reply:
ok, make sense